File: informatykaEuropejczyka/Rozdzial2/prog2_13.cpp
//liniowe przeszukiwanie ciagu liczbowego
#include <iostream>
using namespace std;
const int MAX = 100;

bool szukaj (int T[], int n, int szukana)
{
 int i=0;
 while (i<n && T[i]!=szukana) i++;
 if (i==n) return false;
 return true;
}

int main()
{
 int T[MAX], n, szukana;
 cout<<"podaj liczbe elementow tablicy: ";
 cin>>n;
 cout<<"podaj elementy tablicy:"<<endl;
 for (int i=0;i<n;i++)
 {
  cout<<"T["<<i<<"] = ";
  cin>>T[i];
 }
 cout<<"podaj szukany element: ";
 cin>>szukana;
 if (szukaj(T,n,szukana)) cout<<"TAK"<<endl;
 else cout<<"NIE"<<endl;
 return 0;
}

File: informatykaEuropejczyka/Rozdzial2/prog2_13.py
#liniowe przeszukiwanie ciagu liczbowego

def szukaj (T, szukana):
    i=0
    n=len(T)
    while i<n and T[i]!=szukana:
        i+=1
    if i==n:
        return False
    return True

def szukaj2 (T, szukana):
    if szukana in T:
        print("TAK")
    else:
        print("NIE")

#print(szukaj([7,5,6,4,5,3,4,8,2,3],2))
#szukaj2([7,5,6,4,5,3,4,8,2,3],2)

